test/framework/shoot_utils.go

```diff
@@ -61,13 +60,19 @@ func (f *ShootFramework) GetValiLogs(ctx context.Context, valiLabels map[string]
 
 	command := fmt.Sprintf("wget 'http://localhost:%d/vali/api/v1/query' -O- --post-data='%s'", valiPort, query)
 
-	if isValiAuthEnabled(ctx, log, valiLabelsSelector, valiNamespace, client) {
+	enabled, err := isValiAuthEnabled(ctx, log, valiLabelsSelector, valiNamespace, client)
```

Comment:
Do we still need this code when auth is always disabled now?

Reply:
Why?

Reply:
We can remove it once the vali configs are updated everywhere. Until then it shall remain allowing the `test machinary` test to be executed agains both `old` and `new` configurations.

Reply:
To my knowledge, this TestMachinery code is always executed against Gardener installations with the same version, and only for newly created test clusters (never existing ones).
Is there a scenario where this (newer test machinery code, to be released with g/g@v1.85) is executed against a Gardener installation < v1.85?
If not, I think we can safely clean this up already now.

Reply:
There is no such guarantee. Or this only holds for CI/CD pipeline. At the moment if a developer run the test against his/hers existing infrastructure, the tests will fail. Hence to not break current development workflow, it is preferable to accommodate the test to work also against < g/g@v1.85.

Reply:
OK, then please at least add a TODO statement to clean this up after v1.88 has been released (we typically keep migration code for ~3 releases) and make a reminder to follow-up after CW6 so that we don't forget about it.